src/clients/Device.js

```diff
@@ -811,15 +813,13 @@ class Device extends EventEmitter {
   };
 
   releaseOwnership = (flasher: Flasher) => {
-    logger.log('releasing flash ownership ', { deviceID: this.getDeviceID() });
+    logger.info({ deviceID: this.getDeviceID() }, 'releasing flash ownership ');
     if (this._owningFlasher === flasher) {
       this._owningFlasher = null;
     } else if (this._owningFlasher) {
       logger.error(
-        'cannot releaseOwnership, ',
-        flasher,
-        " isn't the current owner ",
-        { deviceID: this.getDeviceID() },
+        { deviceID: this.getDeviceID(), flasher },
+        'cannot releaseOwnership, isn\'t  current owner'
```

Comment:
The reason is simple. These are parameters.

```
function (
   a,
   b,
)
```
a trailing comma after 'b' is allowed but NOT forced by the current coding style.

So  configuration of eslint needs to be changed to force usage of trailing commas on function parameters

I will make a PR for this